corehq/db/config.yml

```diff
@@ -0,0 +1,20 @@
+# HQ database configuration for multiple database. This gets mapped to Django's DATABASES dict
```

Comment:
good point, at the time simon and i had thought it would be nice to configure it in a yml file, but it may be best to move it to settings. we need a mapping that allows to know which databases belong to which "groups" and that's not something we can currently do with the database config. we also need to specify which shards live on which database (also something we can't do with the current config). we'll definitely need another sort of dict, but this may not be the best place. @snopoke any thoughts on whether this yml file should stay?